stock_warehouse_calendar/__manifest__.py

```diff
@@ -4,15 +4,15 @@
 {
     "name": "Stock Warehouse Calendar",
     "summary": "Adds a calendar to the Warehouse",
-    "version": "11.0.1.0.0",
-    "license": "AGPL-3",
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
```

Comment:
@mreficent Is it nomal you change licensing between version ?

Reply:
I don't know is it's normal or not. But this suppose a problem for you?

Reply:
For me no, but I wanted to know if it's wanted on your side to change licensing from AGPL on v11 to LGPL on v12 ?

Reply:
Yes, it's a wanted thing.